### DISCUSSION TITLE
Few suggestions

### DISCUSSION THREAD

**Discussion by ogohogo** [charleprr/redditube#22] [Ideas]
Hey,

I really enjoy this library!
I have few suggestions to make it even better!

- Add ability to filter inappropriate words (both in replies and in the title)
- As of now when there's any link in the reply, the tts reads the whole `https://www... ` part. The `https://www` part could be removed.
- Allow us to specify custom output path (Having to go into lib's folder in node_modules is slightly annoying)

And also, is there any point in using authentication in order to gather json data about thread? You could fetch data directly from the json API, without the need of authenticating (eg. `https://www.reddit.com/r/AskReddit/comments/108nrnl/what_is_a_fact_that_scares_you/.json`)

Thanks!

**Comment by charleprr** [comment]
Hey,

Those are great suggestions, thank you!
I was planning on making some updates recently, so I might implement some of them.

Also, I didn't know about the JSON API.
That would make it much simpler to use without the need for authentication, I'll look into that.

Thanks again!
